tools/analyzers/protomarshalcheck/protomarshalcheck.go

```diff
@@ -0,0 +1,106 @@
+package protomarshalcheck
+
+import (
+	"flag"
+	"fmt"
+	"go/ast"
+	"regexp"
+	"strings"
+
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+)
+
+func sliceMap(s []string, f func(value string) string) []string {
```

Comment:
I think there's a function for this in the lo library we import some places

Reply:
Is it named the same? I copied this code from an existing analyzer.

Reply:
https://github.com/samber/lo?tab=readme-ov-file#map